src/ethereum_spec_tools/evm_tools/t8n/__init__.py

```diff
@@ -105,13 +104,15 @@ def __init__(
             trace_memory = getattr(self.options, "trace.memory", False)
             trace_stack = not getattr(self.options, "trace.nostack", False)
             trace_return_data = getattr(self.options, "trace.returndata")
-            trace.evm_trace = partial(
-                evm_trace,
+            self.tracer: Union[BaseEvmTracer, FullEvmTracer] = FullEvmTracer(
```

Comment:
Since `FullEvmTracer` inherits from `BaseEvmTracer`, would this work:

```suggestion
            self.tracer: BaseEvmTracer = FullEvmTracer(
```